packages/sdk/src/internal/payloads/payloads.ts

```diff
@@ -35,7 +33,7 @@ export type PayloadType
 	| 'animation-update'
 	| 'app2engine-rpc'
 	| 'collision-event-raised'
-	| 'create-animation'
+	| 'create-animation-2'
```

Comment:
As I mentioned elsewhere, just reuse `create-animation`. We've so far avoided this sort of versioning.

Reply:
Let's discuss this in the other convo.